src/OrleansRuntime/Catalog/ActivationData.cs

```diff
@@ -362,6 +348,10 @@ public GrainId Identity
 
         public IDictionary<object, object> Items { get; private set; }
 
+        public IGrainLifeCycle LifeCycle => FullLifeCycle;
+
+        internal GrainLifecycle FullLifeCycle { get; }
```

Comment:
several places where Cycle is treated as a separate word. Should always be `Lifecycle`